docs/source/how_to_guides/add_custom_dataset.md

```diff
@@ -1,88 +1,175 @@
-# Add a Custom Dataset
 
-This guide explains how to integrate your own dataset into cz-benchmarks.
+# Add a Custom Dataset Type
```

Comment:
This page describes both adding new Datasets (for existing Dataset types) and adding new Dataset types. So the top-level header should be less-specific. Maybe justs "Add a Custom Dataset", with the understanding the sometimes this involves adding a new type.